sdk/examples/wallet/16_destroy_account_output.rs

```diff
@@ -1,15 +1,15 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! In this example we will try to destroy the first alias there is in the account. This is only possible if possible
-//! foundry outputs have circulating supply of 0.
+//! In this example we will try to destroy the first account output there is in the account. This is only possible if
+//! possible foundry outputs have circulating supply of 0.
 //!
 //! Make sure that `example.stronghold` and `example.walletdb` already exist by
 //! running the `create_account` example!
 //!
 //! Rename `.env.example` to `.env` first, then run the command:
 //! ```sh
-//! cargo run --release --all-features --example destroy_alias
+//! cargo run --release --all-features --example destroy_account
```

Comment:
In the Cargo.toml you added this
```suggestion
//! cargo run --release --all-features --example destroy_account_output
```